_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,290 @@
+"active" and may be implemented with the goal of eventual inclusion into Core-Plans.
+
+  - Fork the [Core Plans Repository](https://github.com/habitat-sh/core-plans)
+  - Copy `RFCs/0000-template.md` to `RFCs/0000-my-feature.md` (where "my-feature" is
+    descriptive. don't assign an RFC number yet).
+  - Fill in the RFC. Put care into the details: RFCs that do not present
+    convincing motivation, demonstrate understanding of the impact of the
+    design, or are disingenuous about the drawbacks or alternatives tend to be
+    poorly-received.
+  - Submit a pull request. As a pull request the RFC will receive design
+    feedback from the larger community, and the author should be prepared to
+    revise it in response.
+  - Each pull request will be labeled with the RFC tag, which
+    will lead to its being triaged by the core-plans maintainers in a future meeting and assigned
+    to one of it's members.
+  - Build consensus and integrate feedback. RFCs that have broad support are
+    much more likely to make progress than those that don't receive any
+    comments. Feel free to reach out to the RFC assignee in particular to get
+    help identifying stakeholders and obstacles.
+  - The maintainers will discuss the RFC pull request, as much as possible in the
+    comment thread of the pull request itself. Offline discussion will be
+    summarized on the pull request comment thread.
+  - RFCs rarely go through this process unchanged, especially as alternatives
+    and drawbacks are shown. You can make edits, big and small, to the RFC to
+    clarify or change the design, but make changes as new commits to the pull
+    request, and leave a comment on the pull request explaining your changes.
+    Specifically, do not squash or rebase commits after they are visible on the
+    pull request.
+    - At some point, a maintainer will propose a "motion for final comment period" 
+    (FCP), along with a *disposition* for the RFC (merge, close, or postpone).
+    - This step is taken when enough of the tradeoffs have been discussed that
+    the subteam is in a position to make a decision. That does not require
+    consensus amongst all participants in the RFC thread (which is usually
+    impossible). However, the argument supporting the disposition on the RFC
+    needs to have already been clearly articulated, and there should not be a
+    strong consensus *against* that position outside of the maintainers. We will
+    use our best judgment in taking this step, and the FCP itself
+    ensures there is ample time and notification for stakeholders to push back
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+    a *summary comment* trying to lay out the current state of the discussion
+    and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, *all* Core-Plans Maintainers must sign off;
```

Comment:
I'm a little unclear on this - what happens if not all maintainers are able to sign off in the time period?

Reply:
There is no sign of time period. The time period is only for the FCP. So effectively an RFC can sit open until all maintainers have time to sign off and then the FCP happens (Whether or not we're planning to merge or just close it.) We can tighten this down with a process that enables short circuiting the time an RFC is allowed to remain open but there isn't one currently.